Automattic/wp-calypso
# Add VideoPress / Videomaker Onboarding Flow

#### Proposed Changes

* Adds a new signup flow at `/start/videopress` for onboarding a user to a new premium site with Videomaker.

<img width="1290" alt="Screen Shot 2022-08-09 at 10 48 38 AM" src="https://user-images.githubusercontent.com/789137/183710388-9a6d7277-6dfa-44f9-863a-0f84c6be905b.png">

#### Testing Instructions

* Visit `/start/videopress`, and create a new user and site.
* You should be taken to the checkout page with the premium plan in the cart.
* After payment, you should be redirected to the site editor with Videomaker enabled.

#### Pre-merge Checklist






PT: pxWta-1pv-p2
